Cinebase/Gateways/Container/MoviesContainer.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  MovieContainer.swift
+//  Cinebase
+//
+//  Created by Gustavo Villar on 5/9/17.
+//  Copyright © 2017 Globant. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+class MoviesContainer {
```

Comment:
I will try to make a Container class protocol so we can implement different kinds of containers in the future like ContainerType and inside define generic methods like:

- load()
-save(items: [T])
-delete(items:[T])
-all()

Other thing is make an initializer with the Realm container  and a convenience init with a differente realm file, we will use differents files for ongoing, favorites, cinemas.  look this example

```
final public class PhotoContainer {
    
    fileprivate var container: Realm
    
    private init(container: Realm) {
        self.container = container
    }
    
    public convenience init(name: String) {
        
        var config = Realm.Configuration()
        config.fileURL = config.fileURL!.deletingLastPathComponent().appendingPathComponent("\(name).realm")
        let container = try! Realm(configuration: config)
        
        self.init(container: container)
    }
    
    public static func temporary() -> PhotoContainer {
        
        var config = Realm.Configuration()
        let temporaryDirectoryURL = URL(fileURLWithPath: NSTemporaryDirectory())
        config.fileURL = temporaryDirectoryURL.appendingPathComponent("\(UUID().uuidString).realm")
        let container = try! Realm(configuration: config)
        
        return PhotoContainer(container: container)
    }

}
```